mobile/examples/super_resolution/android/app/src/main/java/ai/onnxruntime/example/superresolution/MainActivity.kt

```diff
@@ -30,8 +30,11 @@ class MainActivity : AppCompatActivity() {
         outputImage = findViewById(R.id.imageView2);
         superResolutionButton = findViewById(R.id.super_resolution_button)
 
+        val sessionOptions: OrtSession.SessionOptions = OrtSession.SessionOptions()
+        sessionOptions.registerCustomOpLibrary(OrtxPackage.getLibraryPath())
+
         superResolutionButton?.setOnClickListener {
-            performSuperResolution()
+            performSuperResolution(ortEnv.createSession(readModel(), sessionOptions))
```

Comment:
Trying to understand the scoping here: so the createSession() is called when the setOnClickListener is declared?

Reply:
sorry - didn't mean to put the create session here. will move it out of this, keep it in OnCreate() along with the registerCustomop steps.